samples/02_power_users_developers/openstreetmap_exploration.ipynb

```diff
@@ -0,0 +1,1338 @@
+{
```

Comment:
<p>I reviewed this notebook by running it locally in 2 different environments:</p><ul><li> one with <em>arcpy</em> created by cloning the default ArcGIS Pro 2.4.2 environment and then upgrading the <em>arcgis</em> to 1.7 and installing <em>osm_runner</em></li><li>one without <em>arcpy</em> by installing <em>arcgis</em> from the latest source code in the git repo and installing <em>osm_runner</em></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>